tests/integ/modin/index/test_argmax_argmin.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+import pandas as native_pd
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.modin.sql_counter import sql_count_checker
+
+
+@sql_count_checker(query_count=1)
+@pytest.mark.parametrize(
+    "native_index",
+    [
+        native_pd.Index([1, None, 4, 3, 4], name="A A A"),
+        native_pd.Index([4, None, 1, 3, 4, 1]),
+        native_pd.Index([4, None, 1, 3, 4, 1], name="some name"),
+        native_pd.Index([1, 10, 4, 3, 4]),
+    ],
+)
+@pytest.mark.parametrize("func", ["argmax", "argmin"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_argmax_argmin_series(native_index, func, skipna):
+    snow_index = pd.Index(native_index)
```

Comment:
nit: why not just use `eval_snowpark_pandas_result`?

Reply:
Ah that's because the result is always a scalar